src/components/HomePage/Slide/Slide.js

```diff
@@ -19,7 +19,10 @@ const Slide = (props) => {
     <div className={cx(CN, className)}>
       <h1 className={`${CN}__title`}>{title}</h1>
       <p className={`${CN}__description`}>{description}</p>
-      <Button className={`${CN}__button`}>
+      <Button
+        className={`${CN}__button`}
+        onClick={() => {}}
```

Comment:
defaultProps candidate

Reply:
it looks like all the props are required

Reply:
still, it would be much nicer